boards/arm/nucleo_l476rg/nucleo_l476rg_defconfig

```diff
@@ -39,3 +39,7 @@ CONFIG_PWM=y
 
 #enable MPU
 CONFIG_STM32_ARM_MPU_ENABLE=y
+
+#enable i2c
+CONFIG_I2C_1=y
```

Comment:
This is not needed if alredy done in Kconfig.defconfig

Reply:
Reverted to previous code.